broker/thinBroker.go

```diff
@@ -1230,6 +1230,8 @@ func (tb *ThinBroker) UnsubscribeContext(w rest.ResponseWriter, r *rest.Request)
 func (tb *ThinBroker) NotifyContextAvailability(w rest.ResponseWriter, r *rest.Request) {
 	notifyContextAvailabilityReq := NotifyContextAvailabilityRequest{}
```

Comment:
[Major] Code have some gap with respect to development branch. Please do the pull request again.